app/assets/stylesheets/activities.css.scss

```diff
@@ -29,7 +29,7 @@ article {
 
 #rafting-card{
       height: 176px;
-      background: url('raft.jpg') center / cover;
+      background: image-url('raft.jpg') center / cover;
```

Comment:
Line should be indented 2 spaces, but was indented 6 spaces<br>Prefer double-quoted strings